sourcemash/frontend/static/js/views/feed_view.js

```diff
@@ -8,8 +8,13 @@ Sourcemash.Views.FeedView = Sourcemash.Views.ItemsView.extend({
     toggleSubscribed: function() {
         if (this.model.get('subscribed')) {
             this.model.save({'subscribed': false}, {success: this.toast});
+            
+            mixpanel.track("Unsubscribed", { "Feed Title": this.model.get('title') })
```

Comment:
We should still track source for unsubscribe because eventually we'll likely offer unsubscribe from both FeedView and FeedsView.